datacube_alchemist/worker.py

```diff
@@ -184,6 +192,7 @@ def convert_eo_plus(ds) -> DatasetDoc:
         'eo:instrument': ds.metadata.instrument,
         'eo:platform': ds.metadata.platform,
         'landsat:landsat_scene_id': ds.metadata_doc.get('tile_id', '??'),  # Used to find abbreviated instrument id
+        'sentinel:sentinel_tile_id': ds.metadata_doc.get('tile_id', '??'),  # Used to find abbreviated instrument id
```

Comment:
Is this comment correct? Looks like it's getting the `tile_id` not the instrument ID

Reply:
bad comment! [It's removed now]